src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,40 @@
+'use client'
+
+import HomeLogoSquare from '../HomeLogoSquare'
+import Link from 'next/link'
+import styles from './styles.module.css'
+
+export default function Footer() {
+  return (
+    <footer className={styles.footer + ` flex shrink flex-col m-5 items-center space-y-3 text-xs`}>
+      <div>
+        <HomeLogoSquare />
+      </div>
+      <p>GovHack and the GovHack Logo are registered trademarks of GovHack Australia Limited.</p>
+      <p>
+        The GovHack Hackerspace is licensed under the AGPL3. Its source code is available&nbsp;
+        <Link
+          href="https://github.com/govhackaustralia/hackerspace3"
+          target="_blank"
+          className={styles.Link}
+        >
+          on GitHub
+        </Link>
+        .
+      </p>
+      <p>
+        <Link href="/terms-and-conditions" className={styles.Link}>
+          Terms and Conditions
+        </Link>{' '}
+        |&nbsp;
```

Comment:
nitpick, just for consistency
```suggestion
        &nbsp;|&nbsp;
```

Reply:
I meant instead of the {' '}. Just seemed like it was mixing two ways to add a space char

Reply:
Ah okay, yeah that was entirely the formatter, not sure why it chose to do it like that but I'll go ahead and fix it up